src/CannedResponses.js

```diff
@@ -0,0 +1,10 @@
+const cannedResponses = Object.freeze({
+  /* Message to be delivered when Alexa responds with an empty message. This is a valid scenario,
+   for example, when user says "shutup" to Alexa.*/
```

Comment:
Can this comment be more polite? ... when user says "stop" to Alexa...

Reply:
Done.